Resources/Prototypes/Corvax/Reagents/Materials/materials.yml

```
@@ -0,0 +1,8 @@
+- type: material
+  id: SheetPrinter
+  stackEntity: SheetPrinter1
+  name: materials-paper
+  icon: { sprite: Objects/Misc/bureaucracy.rsi, state: paper }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)